Source/Diagnostics/FlushFormats/FlushFormatPlotfile.cpp

```diff
@@ -355,7 +354,7 @@ FlushFormatPlotfile::WriteParticles(const std::string& dir,
         UniformFilter const uniform_filter(particle_diags[i].m_do_uniform_filter,
                                            particle_diags[i].m_uniform_stride);
         ParserFilter parser_filter(particle_diags[i].m_do_parser_filter,
-                                   getParser(particle_diags[i].m_particle_filter_parser),
+                                   compileParser<7>(particle_diags[i].m_particle_filter_parser.get()),
```

Comment:
Is the `7` here accidentally hard-coded?
Do you want to use a `m_nvars` var as in the other locations?

Reply:
It has always been hard-coded.  I can change it.